### PR TITLE
community: Added ADOBE PDF EXTRACT

### PR DESCRIPTION
**Pull request by davidmoserai**
**Description:** Adobe PDF Extract is a service that provides superior performance over other document intelligence services, both in its accuracy and variety of features. Parsing documents based on their layout information is crucial for retrieval augmented generation, especially when trying to achieve production grade performance. I have used this service myself successfully and would like to contribute my code to the world.
**Issue:** Someone raised an issue about this a while ago: https://github.com/langchain-ai/langchain/issues/8163
**Dependencies:** A user would have to install the adobe pdf services library like so: pip install pdfservices-sdk

**Twitter handle:** @DavidMoserAI

### CODE REVIEW

**Comment by DavidMoserAI** [comment]
@hwchase17

**Comment by DavidMoserAI** [comment]
@baskaryan @efriis @eyurtsev

File: libs/community/langchain_community/document_loaders/parsers/adobe_pdf_extract.py [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+from langchain_core.documents import Document
+
+if TYPE_CHECKING:
+    from langchain_core.documents.base import Blob
+
+
+class AdobePDFExtractParser(BaseBlobParser):
+    """Loads a document using the Adobe PDF Services API.
+
+    Args:
```
Comment (baskaryan) [inline libs/community/langchain_community/document_loaders/parsers/adobe_pdf_extract.py:18]: could we expand the arg descriptions here

**Comment by davemaguire** [comment]
Hello! I would love to see the adobe pdf api added to langchain. What needs to be done to get this into main? Just address the following comment?

>could we expand the arg descriptions here

I am happy to help get this over the line. We use adobe pdf extract extensively and would love to have this integrated in langchain.
@baskaryan @DavidMoserAI

**Comment by DavidMoserAI** [comment]
@davemaguire I have updated the arg descriptions and am waiting for a response from the moderators.

**Comment by davemaguire** [comment]
Awesome, great work! I'm eager to see this feature in langchain 
@efriis @eyurtsev @hwchase17

**Review by efriis — DISMISSED** [review]

**Comment by efriis** [comment]
@DavidMoserAI this PR had a lot of issues to fix (wrong sdk used in docs, tests for an old Loader instead of Parser implementation), so I'm hesitant to merge it without someone testing it. if you could take at the docs and also screenshot using it to parse an actual PDF using the service, that would be great! Otherwise will probably close without an actual test.

If you're interested in maintaining this integration without us in the loop and publishing a higher-quality integration, we'd love to get an integration package out! Future PRs against langchain would just be {docs updates, as well as registering your package in `libs/packages.yml`, deprecating this community integration in favor of your integration package}

Here's the guide, and if you have questions, feel free to leave them in the comments on those pages so others can see them! https://python.langchain.com/docs/contributing/how_to/integrations/

**Comment by efriis** [comment]
closing for now, and if you decide to pick it up again would recommend publishing externally!

**Pull request closed without merging**